app/src/controllers/testItem/sagas.js

```diff
@@ -118,10 +125,10 @@ function* fetchTestItems({ payload = {} }) {
   );
   const dataPayload = yield take(createFetchPredicate(NAMESPACE));
   let level;
-  if (!parentItem) {
```

Comment:
Why did you revert this change?

Reply:
This change is no longer necessary, SUITE_LEVEL defined as default, also if we go to log page from statistic - level changing to LEVEL_NOT_FOUND which broke navigation logic on logPage.